Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1145,7 +1146,12 @@ class BrowserViewController: UIViewController {
     func restoreSpoofedUserAgentIfRequired(_ webView: WKWebView, newRequest: URLRequest) {
         // Restore any non-default UA from the request's header
         let ua = newRequest.value(forHTTPHeaderField: "User-Agent")
-        webView.customUserAgent = ua != UserAgent.defaultUserAgent() ? ua : nil
+        
+        if #available(iOS 13.0, *) {
+            webView.customUserAgent = Preferences.General.alwaysRequestDesktopSite.value == UserAgent.isDesktopUA(ua) ? nil : ua
+        } else {
+            webView.customUserAgent = ua != UserAgent.defaultUserAgent() ? ua : nil
```

Comment:
I know this was copied from what existed before, but this looks confusing with reversed logic from above, let's modify this so the conditionals look more identical:
```suggestion
            webView.customUserAgent = ua == UserAgent.defaultUserAgent() ? nil : ua
```

Reply:
Fixed.